packages/fast-components-class-name-contracts-base/src/hypertext/index.ts

```diff
@@ -2,5 +2,8 @@
  * The class name contract for the hypertext component
  */
 export interface IHypertextClassNameContract {
+    /**
+     * The root hypertext component
```

Comment:
I think we should tweak the description for all of these "root" classes. Its the root *of* the component, not the root component